src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/integration/CourseIntegrationTest.kt

```diff
@@ -373,6 +375,61 @@ class CourseIntegrationTest : IntegrationTestBase() {
     updatedCourse.withdrawn shouldBe true
   }
 
+  @Test
+  fun `Create course is successful`() {
+    val courseName = "Legacy Course One"
+    val identifier = "LC1"
+    val description = "Test description for Legacy Course"
+    val audience = "Kaizen"
```

Comment:
Audience - this would be something like 'General Offence' but I think this should be the audienceId (after calling the list of audiences)